tark-refseq-loader/handlers/refseq/databasehandler.py

```diff
@@ -437,3 +437,16 @@ def insert_data(self, insert_sql, insert_data, FOREIGN_KEY_CHECKS=1):
             exit(0)
 
         return row_id
+
+    # function to update UTR checksum in translation table; called from parse_gff_file.py
+    def update_utr_checksum(self):
+ 
+        try:
+            connection_pool = self.dbc
+            cursor = connection_pool.cursor()
+            cursor.execute("UPDATE translation tl INNER JOIN translation_transcript tt ON tt.translation_id = tl.translation_id INNER JOIN (SELECT max(transcript_id) as transcript_id FROM transcript GROUP BY stable_id,assembly_id) AS v0 ON v0.transcript_id = tt.transcript_id INNER JOIN transcript t ON tt.transcript_id = t.transcript_id INNER JOIN sequence s ON s.seq_checksum = t.seq_checksum  SET tl.five_utr_checksum = UNHEX(SHA1(IF(t.loc_strand = 1 AND tl.loc_strand = 1, SUBSTRING(s.sequence,1,tl.loc_start-t.loc_start), SUBSTRING(s.sequence,1,t.loc_end-tl.loc_end)))), tl.three_utr_checksum = UNHEX(SHA1(IF(t.loc_strand = 1 AND tl.loc_strand = 1, SUBSTRING(s.sequence,-(t.loc_end-tl.loc_end)), SUBSTRING(s.sequence,-(tl.loc_start-t.loc_start)))))")
```

Comment:
@ranjit-s you can get the checksums using the [checksum handlers](https://github.com/Ensembl/tark-refseq-loader/blob/master/tark-refseq-loader/handlers/refseq/checksumhandler.py) instead of generating them here using UNHEX

Reply:
checksums are now updated using the checksum handlers instead of generating them using UNHEX